garden/migration.md

```diff
@@ -0,0 +1,465 @@
+- Environment variables (names and values)
+    - e.g. `IGN_GAZEBO_RESOURCE_PATH` → `GZ_SIM_RESOURCE_PATH`
+- Command line
+    - e.g. `ign` → `gz`, `ign gazebo` → `gz sim`
+- GUI QML
+    - e.g. `IgnSpinBox` → `GzSpinBox`
+- Topics* (typically in tests)
+    - e.g. `/ignition/XXX` → `/gz/XXX`
+    - **Note:** `/gazebo` is NOT migrated to `/sim`
+- GitHub organizations and repositories
+    - e.g. `ignitionrobotics` → `gazebosim` , `ign-cmake` → `gz-cmake`
+- GitHub actions and workflows
+    - e.g. `ignition-tooling` → `gazebo-tooling`
+- Websites
+    - e.g. `[ignitionrobotics.org](http://ignitionrobotics.org)` → `gazebosim.org`
+- SDF and launch tags
+    - e.g. `<ignition-gui>` → `<gz-gui>`
+
+</details>
+
+### Hard-tocks
+
+There are some exceptions that have been hard-tocked instead, meaning that you **MUST** use the Gazebo counterpart. Using the Ignition counterpart will likely cause compilation or something else to break (unless it’s just a documentation change.)
+
+<details>
+<summary>Hard-tocks</summary>
+
+**Namespaces**
+
+- Ruby namespaces
+    - e.g. `ignition/math` → `gz/math`
+
+**Source**
+
+- C++ Macros in private headers (typically something in a `detail/` directory)
+    - e.g. `DETAIL_IGNITION_UTILS_WARN_RESUME` → `DETAIL_GZ_UTILS_WARN_RESUME`
+- Class names, members, functions, and variables in private headers, or function-local scope
+    - e.g. `kIgnAuthDomain` → `kGzAuthDomain`
+- Install space
+    - e.g. `install/share/ignition` → `install/share/gz`
+- Header guards (where appropriate)
+    - e.g. `IGNITION_COMMON_MATERIAL_HH_` → `GZ_COMMON_MATERIAL_HH_`
+
+**CMake and Packaging**
+
+- Most includeable CMake files in `gz-cmake`
+    - e.g. `IgnUtils.cmake` → `GzUtils.cmake`
+- Gazebo library CMake project names
+    - e.g. `ignition-utils2` → `gz-utils2`
+- Internally used CMake variables
+    - e.g. `IGN_TRANSPORT_VER` → `GZ_TRANSPORT_VER`
+
+**Misc.**
+
+- Gazebo launchfiles and tags
+    - e.g. `sim.ign` → `sim.gzlaunch`, `<ign`  → `<gz`
+- Config and log paths (PENDING)
+    - e.g. `~/.ignition/gui/log` → `~/.gz/gui/log`
+- Fuel URL (PENDING)
+    - e.g. https://fuel.ignitionrobotics.org
+- Fuel cache paths (ALSO PENDING)
+    - e.g. `~/.ignition/fuel` → `~/.gz/fuel`
+- Library names in documentation and comments
+    - e.g. Ignition Gazebo → Gazebo Sim
+- `gz-launch` Websocket server
+    - e.g. `ign.js` → `gz.js`
+
+**Additionally**
+
+- `gz-common` plugins are deprecated, use `gz-plugin` instead
+- `gz-common` `SuppressWarning` is deprecated, use `gz-utils` instead
+
+Also, anything that is internal to the core Gazebo libraries and not used in downstream libraries (e.g. header guards, private headers or source, tests, documentation) is hard-tocked.
+
+</details>
+
+## Untocks
+
+A very small selection of things have not been migrated, mostly for backwards compatibility reasons (e.g. supporting Fortress.)
+
+- Branch names for versions of Gazebo libraries targeting releases before Garden
+    - e.g. `ign-cmake1`
```

Comment:
Nit: `ign-cmake1` has never been part of a collection, we use `ign-cmake2` since Acropolis

```suggestion
    - e.g. `ign-cmake2`
```